packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -536,23 +537,23 @@ test.serial('scenario: Flow 1', async t => {
         locked: { value: scale6(outcome.vaultsActual[1].locked) },
       },
     );
-    t.like(
-      readLatest('published.vaultFactory.managers.manager0.vaults.vault2'),
-      {
-        debt: undefined,
-        vaultState: 'liquidated',
-        locked: { value: scale6(outcome.vaultsActual[2].locked) },
-      },
-    );
+    // t.like(
+    //   readLatest('published.vaultFactory.managers.manager0.vaults.vault2'),
+    //   {
+    //     debt: undefined,
+    //     vaultState: 'liquidated',
+    //     locked: { value: scale6(outcome.vaultsActual[2].locked) },
+    //   },
+    // );
   }
 
   // check reserve balances
-  t.like(readLatest('published.reserve.metrics'), {
-    allocations: {
-      ATOM: { value: scale6(outcome.reserve.allocations.ATOM) },
-      // not part of product spec
-      Fee: { value: scale6(1.54) },
-    },
-    shortfallBalance: { value: scale6(outcome.reserve.shortfall) },
-  });
+  // t.like(readLatest('published.reserve.metrics'), {
+  //   allocations: {
+  //     ATOM: { value: scale6(outcome.reserve.allocations.ATOM) },
+  //     // not part of product spec
+  //     Fee: { value: scale6(1.54) },
+  //   },
+  //   shortfallBalance: { value: scale6(outcome.reserve.shortfall) },
+  // });
```

Comment:
Why are these commented out? Should they be revived, or the reason documented?

Reply:
oops this file wasn't supposed to be included. it was renamed to `test-liquidation-1.js`. I've removed this in an amendment